easybuild/easyconfigs/t/Tk/Tk-8.6.8-GCCcore-6.4.0.eb

```diff
@@ -26,6 +26,9 @@ dependencies = [
 
 configopts = '--enable-threads --with-tcl=$EBROOTTCL/lib CFLAGS="-I$EBROOTTCL/include"'
 
+version_major_minor = version.split('.')[:2]
```

Comment:
@mstud You don't need this line, `%(version_major_minor)s` is a template supported by EasyBuild itself.